dht22_exporter.py

```diff
@@ -40,16 +40,21 @@ def read_sensor(pin):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-g", "--gpio", dest="gpio", type=int, 
+    parser.add_argument("-g", "--gpio", dest="gpio", type=int,
                         required=True, help="GPIO pin number on which the sensor is plugged in")
-    parser.add_argument("-i", "--interval", dest="interval", type=int, 
+    parser.add_argument("-a", "--address", dest="addr", type=str, default=None,
```

Comment:
We could be super safe and just assign the default to `127.0.0.1` making the user have to explicitly say they want a different exposed address. :thinking: